minio/minio
# send backendInfo as part AccountInfo

## Description
send backendInfo as part AccountInfo

## Motivation and Context
This is needed for console UI to change
the UI behavior for different modes of
operation.

## How to test this PR?
Nothing special AccountInfo now has an additional
field to check for server type.

## Types of changes

## Checklist:
